pkg/tnf/identifier/identifier_test.go

```diff
@@ -83,3 +83,41 @@ func TestIdentifier_UnmarshalJSON(t *testing.T) {
 		}
 	}
 }
+
+func TestXformToGinkgoItIdentifier(t *testing.T) {
```

Comment:
This works but you could potentially shorten/encompass all test cases in something like:

```
testCases := []struct{}{}

for _, tc := range testCases { } 
```

Reply:
Ok, I have combined both wrong and ok test, including in the tests structure the expected result of each one.